src/test/java/tutorease/address/logic/commands/DeleteLessonCommandTest.java

```diff
@@ -74,6 +74,32 @@ public void constructor_nullIndex_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> new DeleteLessonCommand(null));
     }
 
+    @Test
+    public void equals() throws CommandException, ParseException {
+        Index index = Index.fromZeroBased(0);
+        DeleteLessonCommand deleteLessonCommand = new DeleteLessonCommand(index);
+        // Test equality with the same object (identity)
+        assertTrue(deleteLessonCommand.equals(deleteLessonCommand));
+
+        // Test equality with null
+        assertFalse(deleteLessonCommand.equals(null));
+
+        // Test equality with an object of different type
+        assertFalse(deleteLessonCommand.equals(new Object()));
+
+        // Test equality with a different DeleteLessonCommand (different index)
+        DeleteLessonCommand differentCommand = new DeleteLessonCommand(Index.fromOneBased(2));
+        assertFalse(deleteLessonCommand.equals(differentCommand));
+    }
+
+    @Test
+    public void toString_correctFormat() {
+        DeleteLessonCommand deleteLessonCommand = new DeleteLessonCommand(Index.fromOneBased(1));
+        // Verify that the toString method returns the correct format
+        assertEquals("DeleteLessonCommand[targetIndex=tutorease.address.commons.core."
+                + "index.Index{zeroBasedIndex=0}]", deleteLessonCommand.toString());
```

Comment:
Since delete lesson uses the same string format, would it be possible to extract the format as a constant which will be used for both toString and for comparison here? Then you would be able to use the string format here to compare
For example
```
String.format(DELETE_COMMAND_STRING_FORMAT, 0)
```
Therefore if the toString() in DeleteLessonCommand is changed we wont have to update everything else.

Reply:
Oh yeah, I think I have hard coded in such way somewhere else. Trying to find it.

Reply:
Fixed!